ludwig/features/vector_feature.py

```diff
@@ -41,10 +41,32 @@
 from ludwig.utils import output_feature_utils
 from ludwig.utils.misc_utils import set_default_value
 from ludwig.utils.torch_utils import LudwigModule
+from ludwig.utils.types import TorchscriptPreprocessingInput
 
 logger = logging.getLogger(__name__)
 
 
+class _VectorPreprocessing(torch.nn.Module):
+    def forward(self, v: TorchscriptPreprocessingInput):
```

Comment:
nit: add return type? `torch.Tensor`?

Reply:
Added to this function as well as the other torchscript preprocessing modules 🙂